main.py

```diff
@@ -10,21 +9,18 @@
 import netCDF4  # for strange reasons, one has to import netCDF4 before geopyspark on some systems, otherwise reading nc-files fails.
 import geopyspark as gps
 from pyspark import SparkContext
-from dotenv import load_dotenv
-
+from metadata_schema import *
 from geoPy import geopy
 
-load_dotenv()
+
 BACKEND_SLURM = 'slurm'
 BACKEND_PYSPARK = 'pyspark'
 BACKEND = os.getenv('BACKEND')
 if BACKEND == BACKEND_SLURM:
     SSH_KEYFILE_PATH = os.getenv('SSH_KEYFILE_PATH')
     SSH_USER_NAME = os.getenv('SSH_USER_NAME')
 
-app = Flask('cuizinart')
```

Comment:
Why do we no longer initialize the app in the main file? We might need `app` in contexts other than the database, so wouldn't it be cleaner to have it here and import it where needed?

Reply:
There's a circular reference issue when importing `metadata_schema.py` before, the `main.py` still has the `app` in context.

Reply:
Okay. Le's leave it that way then.